package.json

```diff
@@ -15,7 +15,8 @@
     "test:coverage": "jest --collectCoverage --coverageDirectory=\"./coverage\"",
     "build:dev": "tsc",
     "build:prod": "tsc -p tsconfig.prod.json",
-    "prepublishOnly": "npm install && npm run build:prod"
+    "prepublishOnly": "npm install && npm run build:prod",
+    "prepare": "npm run build:prod"
```

Comment:
why do we need it?

Reply:
Without prepare script I was not able to get _dist_ folder while installing my fork as:

`npm install git@github.com:BogdanGorelkin/react-native-otp-entry.git`